optimize/backend/src/it/java/io/camunda/optimize/test/it/extension/db/ElasticsearchDatabaseTestService.java

```diff
@@ -620,6 +524,49 @@ public void updateUserTaskDurations(
     getOptimizeElasticClient().update(update);
   }
 
+  @Override
+  public boolean indexExists(final String indexOrAliasName) {
+    final GetIndexRequest request = new GetIndexRequest(indexOrAliasName);
+    try {
+      return getOptimizeElasticClient().exists(request);
+    } catch (final IOException e) {
+      final String message =
+          String.format(
+              "Could not check if [%s] index already exist.", String.join(",", indexOrAliasName));
+      throw new OptimizeRuntimeException(message, e);
+    }
+  }
+
+  @Override
+  @SneakyThrows
+  public boolean zeebeIndexExists(final String indexName) {
```

Comment:
I am not sure I understand this method name, as the implementation doesn't seem zeebe related. Can we check here and if necessary rename it to something more appropriate?

Reply:
The difference seems to be that this one does not add the "optimize" prefix to the indexName before checking if it exists, looking at the usages of both I think they're used correctly so far but the method names are unclear. I will rename to make it more obvious

Reply:
Wdt about `indexExistsCheckWithApplyingOptimizePrefix` and `indexExistsCheckWithoutApplyingOptimizePrefix` ? too wordy? 🤔

Reply:
Ah ok, makes sense. Yeah renames are fine for me - verbosity isn't a bad thing with such wholesale changes 👍